tutorials/printing.adoc

```diff
@@ -1,83 +1,218 @@
-Calibration Set], Public Domain.)
+Adresa tiskárny:: http://printerX.ucebny
 
-[[2-úkol-1-bod]]
-2. úkol (1 bod)
-^^^^^^^^^^^^^^^
+K přihlášení použijte níže uvedené přihlašovací údaje. Heslo sděluje cvičící.
 
-V předchozím úkolu jste již ověřili, že tiskárna tiskne správné rozměry.
-Teď byste rádi vynahradili své drahé polovičce, že jste v práci a
-testujete tiskárny místo toho, abyste byli doma. Proto se rozhodnete
-vytisknout link:../stls/printing/cuteocto.stl[model chobotnice] na
-památku. Bohužel je moc velká a nemáte tolik času. Proto musíte upravit
-rozměry a vytisknout ji o polovinu menší. Kdyby se něco pokazilo,
-vytisknete najednou raději hned 2 kopie. U chobotnice není nutné, aby
-byla dutá, nastavte tedy tisk tak, aby měla výplň alespoň 10%.
-Zajistěte, aby hlava chobotnice nebyla propadlá (použijte dostatečné
-množství perimetrů).
+Jméno:: student
+Heslo:: _sdělí cvičící_
 
-(Model je z https://www.thingiverse.com/thing:27053[Cute Octopus Says
-Hello], CC BY MakerBot.)
+Pokud se na tiskárnu nelze připojit, vyzkoušejte xref:../troubleshooting/printing.adoc#_na-tiskárnu-se-nelze-připojit[jiné možnosti].
 
-[[3-úkol-1-bod-povinný-úkol]]
-3. úkol (1 bod, povinný úkol)
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+CAUTION: Po připojení se ujistěte, že se jedná o rozhraní pro vaši tiskárnu, která vám leží na stole. Ač sabotování tisku spolužákům se může zdát jako legrace, vězte, že není.
 
-Ukliďte po sobě. Kdo po sobě neuklidí, nedostane body.
+
+=== OctoPrint
+
+https://github.com/foosel/OctoPrint[OctoPrint] je jednoduché a přehledné webové rozhraní pro ovládání tiskáren připojených k počítači (v našem případě k Raspberry Pi). Umožňuje tiskárnu ručně ovládat, zobrazit přehledně její stav a odesílat tisková data na tiskárnu.
+
+.Snímek prohlížece s otevřeným OctoPrint rozhraním
+image::../images/printing/octoprint-main.png[OctoPrint]
+
+
+.Ruční ovládání v OctoPrint
+image::../images/printing/octoprint-control.png[Karta ručního ovládání, 250, float="right"]
+==== Ruční ovládání
+
+Na kartě ručního ovládání (Control) lze, jak název napovídá, ručně ovládat tiskárnu.
+Můžete posouvat tiskovou hlavu ve všech třech osách a také ovládat extruder. 
+
+První sloupec je velmi intuitivní, šipky představují směry, do kterých můžete tiskovou hlavu posunout.
+Ikona domečku představuje akci "home", která vyresetuje pozici tiskárny na `[0, 0, 0]` a zároveň sesynchronizuje fyzickou pozici tiskové hlavy s údaji v kontroléru tiskárny.
+
+Tlačítka `0.1`, `1`, `10` a `100` představují hodnotu o kterou posunete tiskovou hlavu následným kliknutím na šipku směru.
+Před posunem tlačítky je však nutné tiskárnu vždy uvést do stavu "home", aby tiskárna nevyjela mimo hranice.
+Pokud byste násilím posouvali osy mimo hranice, můžete tiskárnu poškodit.
+
+V druhém sloupci lze ovládat extruder. 
+V číselném poli lze nastavit kolik milimetrů materiálu bude extrudováno/retractováno a tlačítky pod ním lze příslušnou akci provést.
```

Comment:
~ Tohle se dá dělat, jen když je nahřáto. O teplotách se toho dozvíte více dále.